pyg_lib/sampler/__init__.py

```diff
@@ -4,6 +4,41 @@
 from torch import Tensor
 
 
+def neighbor_sample(
+    rowptr: Tensor, col: Tensor, seed: Tensor, num_neighbors: list[int],
+    replace: bool = False, directed: bool = True, isolated: bool = True,
+    return_edge_id: bool = True
```

Comment:
```suggestion
    return_edge_id: bool = True,
```
and let `yapf` format.

Reply:
Done.